cmake/finds/RequiredDependencySearch.cmake

```diff
@@ -97,14 +97,12 @@ set(MUQ_HAS_SUNDIALS 1) # this is needed for preprocessor directives in the MUQ
 
 find_package(Eigen3 3.3 REQUIRED NO_MODULE)
 
-############################################
-##### LOOK FOR AND/OR BUILD NANOFLANN ######
-############################################
+###############################
+##### LOOK FOR NANOFLANN ######
+###############################
+
+find_package(nanoflann)
```

Comment:
why is this not reuired?

Reply:
oh you're right I missed that, adding the setting.